lib/reddex_web/plugs/auth.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ReddexWeb.Plugs.Auth do
+  @moduledoc "Check if current_user is present in the session"
+
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(conn, _) do
+    # TODO: Not sure about this
+    # maybe be more explicit in controllers with actions
+    if get_session(conn, :current_user) || conn.request_path == "/sign_in" do
```

Comment:
I'd leave it like this for now and refactor later, when more "public" pages are at place.

Reply:
ok